src/vivarium/framework/plugins.py

```diff
@@ -22,14 +22,16 @@
 from vivarium.framework.lifecycle import LifeCycleInterface, LifeCycleManager
 from vivarium.framework.logging.manager import LoggingInterface, LoggingManager
 from vivarium.framework.lookup.manager import LookupTableInterface, LookupTableManager
-from vivarium.framework.population.manager import PopulationInterface, PopulationManager
+from vivarium.framework.population.interface import PopulationInterface
```

Comment:
Should these imports be like this?
```python
from vivarium.framework.population import PopulationInterface, PopulationManager
```

Reply:
Ok, no circular imports and saved two lines 👍